### PR TITLE
Roll alert features and oic changes to production

### PR DESCRIPTION
**Pull request by andrewkrug**
- Adds Firefox out of date alert
- Refactor user model
- Ansible forces installation of patch oic module.

### CODE REVIEW

**Comment by andrewkrug** [comment]
Do not merge.  Approval only.  Will merge at deploy time when CAB has been filed.

**Comment by andrewkrug** [comment]
This needs to merge after #177  and also https://github.com/mozilla-iam/auth0-deploy/pull/147

File: ansible/roles/dashboard/tasks/main.yml [hunk 1]
```diff
@@ -107,3 +106,14 @@
     owner: flaskapp
     group: nginx
     recurse: yes
+
+- name: Install patched flask-pyoidc
```
Comment (gdestuynder) [inline ansible/roles/dashboard/tasks/main.yml]: keep or not keep?

**Comment by andrewkrug** [comment]
CHG0012095 filed in ServiceNow.

**Pull request merged**